app/src/main/kotlin/compiler/ast/StatementAst.kt

```diff
@@ -0,0 +1,34 @@
+package compiler.ast
+
+typealias BlockAst = List<StatementAst>
+
+sealed class StatementAst {
```

Comment:
Sometimens `Comments` are also included in AST, do we want to add them?

Reply:
The compiler probably won't make any use of them, so I don't see a point.